deps/wrapper.cc

```diff
@@ -373,6 +389,7 @@ JLCXX_MODULE define_julia_module(jlcxx::Module& mod)
     mod.method("LocalMemoryBuffer", [] (uint8_t *data, size_t size, bool copy_data = false) {
         return std::make_shared<LocalMemoryBuffer>(data, size, copy_data);
     });
+    jlcxx::stl::apply_stl<std::shared_ptr<Buffer>>(mod);
```

Comment:
Move the `std::vector` support to be next to `Buffer`
```suggestion
    jlcxx::stl::apply_stl<std::shared_ptr<Buffer>>(mod);
    mod.method("LocalMemoryBuffer", [] (uint8_t *data, size_t size, bool copy_data = false) {
        return std::make_shared<LocalMemoryBuffer>(data, size, copy_data);
    });
```